ui/webpack.common.ts

```diff
@@ -33,7 +33,8 @@ module.exports = {
     rules: [
       {
         test: /\.wasm$/,
-        type: 'webassembly/experimental',
+        loader: "file-loader",
```

Comment:
this is required to allow the string tokenizer to work. i couldn't find us using the other wasm file, so i dont think this'll break anything